### PR TITLE
Add deployment step

### PR DESCRIPTION
**Pull request by dmitry-shibanov**
- add release step in pipeline
- add create-release script

### CODE REVIEW

File: images.CI/create-release.ps1 [hunk 1]
```diff
@@ -0,0 +1,33 @@
+param(
+    [String] [Parameter (Mandatory = $True)] $BuildId,
+    [String] [Parameter (Mandatory = $True)] $Organization,
+    [String] [Parameter (Mandatory = $True)] $Project,
+    [String] [Parameter (Mandatory = $True)] $ImageName,
+    [String] [Parameter (Mandatory = $True)] $DefinitionId,
+    [String] [Parameter (Mandatory = $True)] $AccessToken
+)
+
+$Body = @"
```
Comment (maxim-lobanov) [inline images.CI/create-release.ps1]: Why do you use multiline text to create json object?
It is simpler to just create powershell object and convert it to the json

```
$Body = @{
  definitionId = $DefinitionId
  variables = @{
    ImageBuildId = @{
      value = "$BuildId"
    }
    ImageName = @{
      value = "$ImageName"
    }
  }
  isDraft = "false"
} | ConvertTo-Json -Depth 3
```

File: images.CI/azure-pipelines/image-generation.yml [hunk 2]
```diff
@@ -25,6 +25,18 @@ jobs:
                         -VirtualNetworkSubnet $(BUILD_AGENT_SUBNET_NAME) `
                         -GitHubFeedToken $(GITHUB_TOKEN)
 
+  - task: PowerShell@2
+    displayName: 'Deploy VM'
```
Comment (maxim-lobanov) [inline images.CI/azure-pipelines/image-generation.yml]: Create release for VM deployment

Comment (dmitry-shibanov) [reply images.CI/azure-pipelines/image-generation.yml]: Done.

File: images.CI/create-release.ps1 (hunk 1 above)
Comment (dmitry-shibanov) [reply images.CI/create-release.ps1]: Done.

File: images.CI/create-release.ps1 [hunk 3]
```diff
@@ -0,0 +1,31 @@
+param(
+    [UInt32] [Parameter (Mandatory)] $BuildId,
+    [String] [Parameter (Mandatory)] $Organization,
+    [String] [Parameter (Mandatory)] $Project,
+    [String] [Parameter (Mandatory)] $ImageName,
+    [String] [Parameter (Mandatory)] $DefinitionId,
```
Comment (vmapetr) [inline images.CI/create-release.ps1]: Nitpick: definition id also should have int type.

**Pull request merged**